spec/controllers/idv/image_uploads_controller_spec.rb

```diff
@@ -56,34 +56,21 @@
         stub_analytics
         stub_attempts_tracker
 
-        expect(@analytics).to receive(:track_event).with(
-          'IdV: doc auth image upload form submitted,',
-          success: false,
-          errors: {
-            front: ['Please fill in this field.'],
-          },
-          error_details: {
-            front: { blank: true },
-          },
-          user_id: user.uuid,
-          submit_attempts: 1,
-          remaining_submit_attempts: IdentityConfig.store.doc_auth_max_attempts - 1,
-          pii_like_keypaths: pii_like_keypaths,
-          flow_path: 'standard',
-        ).exactly(0).times
```

Comment:
This struck me as slightly odd--"make we receive this _extremely specific_ method call exactly 0 times". Also there was a comma in the event name. I think what this is checking is that if the form validation fails, we are not logging `IdV: doc auth image upload vendor submitted`, so I updated it to that.